use-timescale/compression/modify-compressed-data.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Inserting or modifying data in compressed chunks
+excerpt: What happens when you try to modify data in a compressed hypertable
+products: [cloud, mst, self_hosted]
+keywords: [compression, backfilling, hypertables]
+---
+
+# Inserting into compressed chunks
+
+In TimescaleDB&nbsp;2.11 and later, you can insert data into compressed chunks.
+This works even if the data you are inserting has unique constraints, and those
+constraints are preserved during the insert operation. This is done by using a
+PostgreSQL function that decompresses relevant data on the fly to check if the
+new data breaks unique checks. This means that any time you insert data into a
+compressed chunk, a small amount of data is decompressed to allow a speculative
+insertion, and block any inserts which could violate constraints.
+
+In TimescaleDB&nbsp;2.10, you can insert data into compressed chunks with some
+limitations. The primary limitation is that you can't insert data with unique
+constraints. Additionally, newly inserted data needs to be compressed at the
+same time as the data in the chunk, either by a running recompression policy, or
+by using `recompress_chunk` manually on the chunk.
+
+In TimescaleDB&nbsp;2.9 and earlier, you cannot insert data into compressed chunks.
```

Comment:
Just wondering if the different version information can be best presented in the Tabs <Version 2.11 and later> ,<Version 2.10>, <Version 2.9 and earlier>

Reply:
hrm, that's a good idea. Let me try it and we can see how it looks.